subsys/shell/modules/kernel_service.c

```diff
@@ -88,6 +99,8 @@ static int cmd_kernel_threads(const struct shell *shell,
 	ARG_UNUSED(argc);
 	ARG_UNUSED(argv);
 
+	shell_fprintf(shell, SHELL_NORMAL,
```

Comment:
please check `shell_print` macro. Could be used here (it prints in normal and adds new line).

Reply:
Thats odd. I even missed that it's **f**printf. whole file is using the fprintf/ maybe i will use proper one. Does it make sense to update whole file in separate PR?

Reply:
Seems like i'm too lazy to create a separate PR. Done in the same